### ISSUE TITLE
GroupedList has an accessibility issue regarding ARIA attributes

### ISSUE THREAD

**Issue by ms-kykwon** [microsoft/fluentui#17943]
### Environment Information

- **Package version(s)**: 8.12.0 (official doc site)
- **Browser and OS versions**: Chrome  89.0.4389.128, Windows 10

### Describe the issue:

The GroupedList component has an accessibility issue that complains about not allowing ARIA attributes: `aria-setsize` and `aria-posinset`.

#### Actual behavior:

![image](https://user-images.githubusercontent.com/83098653/115937777-8bcf7c80-a44d-11eb-9f11-9dcf590d2f81.png)

#### Expected behavior:

No accessibility failures.

### Documentation describing expected behavior

https://dequeuniversity.com/rules/axe/3.5/aria-allowed-attr?application=axeAPI

**Comment by ms-kykwon** [comment]
Duplicated issue: https://github.com/microsoft/fluentui/issues/16144

**Comment by smhigley** [comment]
This is a false positive from the automated testing tool, and an issue is already open here: https://github.com/dequelabs/axe-core/issues/2794 (axe-core is the rules engine for the automated testing part of Accessibility Insights)
